doc/omnisharp-vim.txt

```diff
@@ -517,6 +517,19 @@ convenient user re-mapping. These can be used like so: >
     :OmniSharpGotoDefinition vsplit
     :OmniSharpGotoDefinition tabedit
 <
+                                                       *:OmniSharpGotoTypeDefinition*
```

Comment:
Hi @nickspoons, would you help me fix this? It does not display when I type `help OmniSharpGotoTypeDefinition`.

Reply:
Have you checked the response received from the server? When I try it with this PR I'm getting a response like this:

```json
{
  "Request_seq": 1021,
  "Command": "/gototypedefinition",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "Definitions": null
  },
  "Seq": 479,
  "Type": "response"
}
```

Reply:
Oh nvm I tried again in other locations and successfully navigated to type definitions (in metadata)

Reply:
You need to regenerate your helptags, which plugin managers like e.g. vim-plug do automatically after installing updating plugins:

```vim
:helptags ~/.vim/plugged/omnisharp-vim/doc
```

Reply:
Previously it did not work because I was rewriting my history by squashing my commits.

I resolved it by deleting the plugin using `vim +PlugClean` and then re-installing `vim +PlugInstall`.